src/components/TodoForm/TodoForm.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import './TodoForm.scss';
+import usersFromServer from '../../api/users';
+import { getUserById } from '../../services/user';
+import { Todo } from '../../types/Todo';
+import { getNewTodoId } from '../../services/todo';
+
+type Props = {
+  todos: Todo[];
+  onSubmit: (newTodo: Todo) => void;
+};
+
+export const TodoForm: React.FC<Props> = ({ todos, onSubmit }) => {
+  const [title, setTitle] = useState('');
+  const [hasTitleError, setHasTitleError] = useState(false);
+
+  const [userId, setUserId] = useState(0);
+  const [hasUserIdError, setUserIdError] = useState(false);
+
+  const handleTitleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setHasTitleError(false);
+  };
+
+  const handleUserIdChange = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    setUserId(+event.target.value);
+    setUserIdError(false);
+  };
+
+  const handleReset = () => {
```

Comment:
What event does this function handle, `onSubmit`, `onChange` or `onClick`?

Let's rename it `resetForm`